CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
+- Added ability to disable long press to add media into playlist via the “Playlist” settings. ([#3670](https://github.com/brave/brave-ios/issues/3670))
+- Added a new tab tutorial page on startup for Japan region. ([#3389](https://github.com/brave/brave-ios/issues/3389))
+- Updated settings menu to display scrollable modal which will allow for more dynamic actions. ([#3491](https://github.com/brave/brave-ios/issues/3491))
+- Updated Startpage icon under onboarding and search engine settings. ([#3584](https://github.com/brave/brave-ios/issues/3584))
+- [Security] Fixed cookie persistence via download request in private mode as reported on Bugzilla by Muneaki Nishimura. ([#2984](https://github.com/brave/brave-ios/issues/2984))
+- [Security] Fixed cross-site scripting issue in RSS feed as reported on HackerOne by Muneaki Nishimura. ([#3630](https://github.com/brave/brave-ios/issues/3630))
+- Improved readability of text under “Brave Shields & Privacy” settings by reducing font weight. ([#3180](https://github.com/brave/brave-ios/issues/3180))
+- Fixed playlist videos appearing under “Downloaded Videos” via iOS storage settings. ([#3633](https://github.com/brave/brave-ios/issues/3633))
+- Fixed https://www.listennotes.com not being recognized by Playlist. ([#3726](https://github.com/brave/brave-ios/issues/3726))
+- Fixed “Add to Brave Playlist” toast not being displayed on https://odysee.com. ([#3663](https://github.com/brave/brave-ios/issues/3663))
+- Fixed memory leak when restoring tabs. ([#1086](https://github.com/brave/brave-ios/issues/1086))
+- Fixed PIN not being removed from keychain when re-installing Brave. ([#3598](https://github.com/brave/brave-ios/issues/3598))
+- Fixed crash when opening certain PDFs. ([#3572](https://github.com/brave/brave-ios/issues/3572))
+- Fixed audio continuing to play after tab has already been closed in certain cases. ([#3544](https://github.com/brave/brave-ios/issues/3544))
+- Fixed Brave News feed state is retained when source is removed and re-added. ([#3468](https://github.com/brave/brave-ios/issues/3468))
+- Fixed moving between text fields causes search button to be hidden. ([#3454](https://github.com/brave/brave-ios/issues/3454))
+- Fixed “Trackers & ads blocked on this page” tooltip being displayed when tab tray is opened in certain cases. ([#3461](https://github.com/brave/brave-ios/issues/3461))
+- Fixed incorrect link being used when using “Share” in certain cases. ([#3246](https://github.com/brave/brave-ios/issues/3246))
+- Fixed “Cancel” button next to URL bar not being completely dismissed in certain cases. ([#2925](https://github.com/brave/brave-ios/issues/2925))
+- Fixed “History” window using incorrect theme under private tab when empty. ([#3719](https://github.com/brave/brave-ios/issues/3719))
+- Fixed theming issues when changing theme via iOS “Control Center” while ad notification visible. ([#2053](https://github.com/brave/brave-ios/issues/2053))
```

Comment:
Can change this to `Fixed browser theme issue when changing device theme via control centre while an ad notification is shown`. Theming doesn't sound right

Reply:
Done 👍